src/oemof/solph/_models.py

```diff
@@ -446,7 +447,31 @@ def relax_problem(self):
         return self
 
 
+class Filters(dict):
+    def updating(
+        self, filters: dict[str, Callable[[object], bool]]
+    ) -> "Filters":
+        result = Filters(self)
```

Comment:
Why is this needed? Rather than `self.update(filters)` ?

Reply:
I tried to explained that in the corresponding [commit message](https://github.com/oemof/oemof-solph/commit/9182ba5cda5d9ddd45bc2a6b8118e26a87b76808).  I agree though, that both, `updating` and `Filters` could use a docstring which would contain some of what is written in the mentioned commit message. Would you mind giving it a read and if that doesn't answer your questions I'll have a better understanding on what to change once I'll write those docstrings.